bentoml/_internal/bento/build_config.py

```diff
@@ -451,10 +529,10 @@ def write_to_bento(self, bento_fs: "FS", build_ctx: str) -> None:
             # Note: "--generate-hashes" is purposefully not used here because it will
             # break if user includes PyPI package from version control system
             pip_compile_in = bento_fs.getsyspath(
-                fs.path.join(py_folder, "requirements.txt")
+                fs.path.combine(py_folder, "requirements.txt")
```

Comment:
Would be nice to document this type of changes in PR in a comment like this:
---

Combine: Join two paths together.

This is faster than [join()](https://docs.pyfilesystem.org/en/latest/reference/path.html#fs.path.join), but only works when the second path is relative, and there are no back references in either path.

https://docs.pyfilesystem.org/en/latest/reference/path.html#fs.path.combine